lib/web_ui/lib/src/engine/navigation/history.dart

```diff
@@ -4,18 +4,18 @@
 
 import 'package:meta/meta.dart';
 import 'package:ui/ui.dart' as ui;
+import '../../../ui_web/src/ui_web.dart' as ui_web;
```

Comment:
How hard would it be to make this similar to the `package:ui/ui.dart` above? (i.e. `package:ui_web/ui_web.dart` or something like that). I guess it's just a matter of rewriting it in the sdk_rewriter?

Reply:
This just gets erased by the rewriter, and in the top-level library file it says `import 'dart:ui_web' as ui_web;` so for the original source files we can import it any way we like as long as it is imported under the name `ui_web`. I can try to find a cleaner way to do this import.

Reply:
On second look, I think `package:ui/src/ui_web.dart` should work already?